airflow/templates/services.yaml

```diff
@@ -1,67 +1,67 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Values.airflow.prefix -}}postgres
+  name: {{ template "airflow.fullname" . }}-postgres
 spec:
   type: ClusterIP
   selector:
     app: airflow
     tier: db
   ports:
-    - name: {{ .Values.airflow.prefix -}}postgres
+    - name: postgres
       protocol: TCP
       port: 5432
-      targetPort: {{ .Values.airflow.prefix -}}postgres
+      targetPort: postgres
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Values.airflow.prefix -}}rabbitmq
+  name: {{ template "airflow.fullname" . }}-rabbitmq
 spec:
   type: ClusterIP
   selector:
     app: airflow
     tier: rabbitmq
   ports:
-    - name: {{ .Values.airflow.prefix -}}node
+    - name: rabbitmq
```

Comment:
please keep the prefix, I need it to allow several airflow in the same namespace

Reply:
I understand. The issue I ran into what the Release Name was somewhat long and combined with the service, it surpassed the 15 character limit.

Reply:
yes it needs to be small and dns compatible